### PR TITLE
Add "Namespaces" paragraph to "Code Style" in `DEVELOPMENT.md`.

### PR DESCRIPTION
**Pull request by dpallotti**

### CODE REVIEW

**Review by dimitry- — APPROVED** [review]

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -221,6 +221,16 @@ with no exceptions. Particularly we don't use methods from std that throw
 exceptions but prefer the variants returning error codes (e.g. in 
 `std::filesystem`).
 
+### Namespaces
+We place all code belonging to a module named `ModuleName` or `OrbitModuleName`
+inside the top-level namespace `orbit_module_name`. We do not use nested
+namespaces.
```
Comment (ronaldfw) [inline DEVELOPMENT.md:227]: Maybe as further clarification, add something like:

"All namespaces start with the `orbit_` prefix."

**Review by ronaldfw — APPROVED** [review]
LGTM

File: DEVELOPMENT.md (hunk 1 above)
Comment (dpallotti) [reply DEVELOPMENT.md]: Yep, done.

**Review by danielfenner — APPROVED** [review]
lgtm!

**Review by beckerhe — APPROVED** [review]
Thanks!

**Pull request merged**